packages/@aws-cdk-testing/framework-integ/test/aws-ecs/test/integ.fargate-task-def.ts

```diff
@@ -0,0 +1,26 @@
+import * as cdk from 'aws-cdk-lib';
+import * as ecs from 'aws-cdk-lib/aws-ecs';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-ecs-fargate-task-def');
+
+const taskDefinition = new ecs.TaskDefinition(stack, 'TaskDef', {
+  compatibility: ecs.Compatibility.EC2_AND_FARGATE,
+  cpu: '256',
+  memoryMiB: '512',
+});
```

Comment:
Since these changes will also affect `ecs.FargateTaskDefinition`, can we include it in an integration test as well? Either in this test or as a separate file should be fine, since we're testing both the regular and Fargate tasks in unit tests already.

Reply:
I have added one more integ test for TaskDefinition